socorro/external/postgresql/bugs.py

```diff
@@ -40,23 +40,21 @@ def get(self, **kwargs):
             sql_params.append(tuple(params.signatures))
 
             sql = """/* socorro.external.postgresql.bugs.Bugs.get */
-                SELECT ba.signature, bugs.id
-                FROM bugs
-                    JOIN bug_associations AS ba ON bugs.id = ba.bug_id
+                SELECT signature, bug_id as id
+                FROM bug_associations
                 WHERE EXISTS(
-                    SELECT 1 FROM bug_associations
-                    WHERE bug_associations.bug_id = bugs.id
+                    SELECT 1 FROM bug_associations as ba
+                    WHERE ba.bug_id = bug_associations.bug_id
```

Comment:
I'm not sure I understand this. I think you don't need the join anymore because you're only looking at a single table. Maybe this is better?:

```
SELECT 1 from bug_associations
WHERE signature IN %s
```

Does that sound right?